server/worker/tasks/send_email.ts

```diff
@@ -54,22 +52,22 @@ const task: Task = async (inPayload) => {
   try {
     const info = await transporter.sendMail(options);
   } catch (e) {
-    console.log(e)
+    console.log(e);
   }
-}
+};
 
 export default task;
 
 const templatePromises = {};
-function loadTemplate(template: string) {
+function loadTemplate(template: string): Promise {
```

Comment:
this is a pretty weak statement, and i'm not sure that it is even valid here.

Reply:
you can't just return a `Promise` like this:
![image](https://user-images.githubusercontent.com/3149358/157395920-d4d0215d-aaf5-44c9-b1fb-f27db6250209.png)
You might want to add a eslint plugin to your IDE if you don't have that already.

You need to add the corresponding returned value which is a function in this case:

```suggestion
function loadTemplate(template: string): Promise<(variables: { [varName: string]: any }) => Promise<string>> {
```

Reply:
i was pretty sure this was not a valid return, really just wanted to get something here for review. i added types for other promises in our code, but this was one of the more complicated return values i came across. thanks for the help! i’ll try to find a plugin that helps with this.